library/core/src/char/methods.rs

```diff
@@ -1088,6 +1090,7 @@ impl char {
     /// [`make_ascii_uppercase()`]: #method.make_ascii_uppercase
     /// [`to_uppercase()`]: #method.to_uppercase
     #[stable(feature = "ascii_methods_on_intrinsics", since = "1.23.0")]
+    #[doc(alias = "toupper")]
```

Comment:
I intentionally chose to use the `_ascii_` variants to match C.

Reply:
I can see the reasoning, but just in case: the C functions depend on the locale.

Reply:
Hmm, ok. Do you think I should add this to `to_uppercase` instead, then?

Reply:
No, I think it is definitely better to have it in the ASCII ones even if it does not match exactly (at least the ASCII interface resembles the C one, i.e. it returns a single "char" rather than several/an iterator). I was mentioning the locale just in case.